packages/core/tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "outDir": "./dist",
     "declaration": true,
     "target": "ES2019",
-    "lib": ["ES2020"]
+    "lib": ["ES2020"],
+    "jsx": "react-jsx"
```

Comment:
Could this be removed now that we don't go for tsx anymore?

Reply:
good catch!